CMakeLists.txt

```diff
@@ -54,6 +54,14 @@ else(Trilinos_BUILD_SHARED_LIBS)
   set(Nalu_LIBRARY_TYPE STATIC)
 endif(Trilinos_BUILD_SHARED_LIBS)
 
+############################ BOOST #####################################
+find_library(BOOST_LIBRARIES NAMES boost_filesystem PATHS ${BOOST_DIR}/lib)
```

Comment:
Can you please use https://cmake.org/cmake/help/latest/module/FindBoost.html instead?

Reply:
Thanks for pointing me to `FindBoost`.  I modified `CMakeLists.txt` to use that instead.